### DISCUSSION TITLE
Replace Image

### DISCUSSION THREAD

**Discussion by andresclua** [sanity-io/sanity-plugin-media#182] [Ideas]
Hey! love your plugin.

Is it a possibility to introduce a feature that allows for asset swapping? Is this something that could be achieved?

This concept was proposed by our marketing team within my organization. They are keen on reducing the dependency with developers.

**Comment by aleczratiu** [comment]
Up. Any updates for this feature?

**Comment by Polleke007** [comment]
https://github.com/sanity-io/sanity-plugin-media/issues/166#issuecomment-2627008615
